tests/test_configuration.py

```diff
@@ -123,7 +123,7 @@ def test_only_service_name_and_token(self, mock_otlp_span_exporter):
         # The worker thread in MockBatchSpanProcessor is configured to
         # wait 1ms before exporting. Sleeping here to give enough time to the
```

Comment:
the comment should be updated to reflect the change

Reply:
Fixed